packages/snap-preact/components/src/components/Molecules/LoadMore/LoadMore.tsx

```diff
@@ -29,6 +29,14 @@ const CSS = {
 		const radialAngle = Math.max(3.6, ((360 / 100) * Math.floor((pagination!.end / pagination!.totalResults) * 100)) / 2);
 
 		return css({
+			'& .ss__load-more__button--disabled': {
+				opacity: 0.7,
+				borderColor: 'rgba(51,51,51,0.7)',
+				backgroundColor: 'initial',
```

Comment:
lets add `pointer-events: none;`

Reply:
Shouldn't the `Button` component be handling these styles via it's disabled state?

Reply:
thats how it used to be, unfortunately the disabled button state would cause the screen reader to update and read that the button was disabled after you clicked it while it was loading. It was a very weird experience. So I opted to keep the button enabled to make the screen reader experience better and just add the disabled styling instead.